RoadMap_Shop/RoadMap_Shop/ProductViewController.swift

```diff
@@ -28,12 +26,24 @@ final class ProductViewController: UIViewController {
         return view
     }()
     
+    private lazy var deliveryOptionsLabel: UILabel = {
+       let label = UILabel()
+        label.text = Constants.TextForUIElements.deliveryOptionsLabelText
+        label.textColor = .systemBlue
+        label.font = .systemFont(ofSize: 10)
+        label.frame = CGRect(x: 40, y: 735, width: 300, height: 25)
+        return label
+    }()
+    
+    var product: Product?
+ 
     // MARK: - Private properties
     private let constants = Constants()
```

Comment:
```suggestion
    private let constants = Constants()
```
зачем объект для констант если можно свойства у констант стариком сделать